styles.css

```diff
@@ -93,7 +101,306 @@ ul.navbar-items {
   line-height: 1.5rem;
 }
 
-@media only screen and (max-width: 375px) {
+/* Works section styling */
+
+#works-section {
+  margin: 10px 0;
+  overflow-y: hidden;
+}
+
+.box-container {
+  display: flex;
+  margin: 50px 200px 50px 200px;
+}
+
+.left-section {
+  flex: 1;
+}
+
+.left-section img {
+  float: left;
+  margin-right: 30px;
+  max-width: 100%;
+}
+
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
+  gap: 10px;
+  margin: 0 200px;
+}
+
+.grid-item {
+  position: relative;
+  background-color: #eaeaea;
+  height: 400px;
+}
+
+.first-desc h3 {
+  color: #091e42 !important;
+  font-size: 32px !important;
+  line-height: 44px !important;
+}
+
+.first-desc p {
+  font-size: 16px !important;
+  line-height: 25px;
+  text-align: left;
+}
+
+.first-desc ul li {
+  display: inline-block;
+  list-style: none;
+  border: 1px solid grey;
+  padding: 10px;
+  margin-right: 10px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.first-desc button {
+  background-color: #f55800;
+  border: none;
+  padding: 10px 10px;
+  margin-top: -40px !important;
+  font-family: 'Crete Round', sans-serif !important;
+  font-style: normal !important;
+  font-weight: 700px !important;
+  font-size: 16px !important;
+  line-height: 30px !important;
+  letter-spacing: 0.03em;
+}
```

Comment:
- [ ] Kindly implement the appropriate interaction to your button. Because it would enhance the user experience 🎲🎰🎯
Below is the design that you should follow for all the **interactions** of your **ALL** buttons 👇👇

![Screenshot from 2023-06-03 07-52-25](https://github.com/Xghayor/Portfolio/assets/10439283/57cf44f6-9e70-4de6-8fab-5a24bfc1b5f0)